content/06.methods.md

```diff
@@ -570,8 +565,14 @@ We predicted telomerase activity of tumor samples using the recently developed E
 In brief, EXTEND estimates telomerase activity based on the expression of a 13-gene signature. 
 We derived this signature by comparing telomerase-positive tumors and tumors with activated alternative lengthening of telomeres pathway, a group presumably negative of telomerase activity. 
 
-### ADDITIONAL RESOURCES
-<!-- TODO: Add section -->
+### Survival models
+
+We calculated overall survival (OS), as days since initial diagnosis (`OS_days`) and progression-free survival (PFS), as days since last event (`PFS_days`).
+Using OS, we visualized survival curves for patients by HGG subtype using the Kaplan-Meier estimator [@doi:10.2307/2281868] and performed a log-rank test (Mantel-Cox test) [@pmid:5910392] of OS between the different HGG subtypes.
```

Comment:
Rather than focus on "we visualized," instead we "PERFORMED" these analyses. I would also note somewhere that we used the R package survival for these.

Reply:
✅